site_src/gambling-stats.ts

```
@@ -0,0 +1,99 @@
+import type { Silverwolf } from '../classes/silverwolf';
+import { formatDisplay } from '../utils/math';
+import { numSpan } from './format';
+import { html } from 'hono/html';
```

ESLint

[error] `hono/html` import should occur before type import of `../classes/silverwolf`

(import/order)